lib/src/duration/duration_extractor.dart

```diff
@@ -577,7 +577,7 @@ class EnglishDurationExtractorConfiguration extends BaseOptionsConfiguration
       RegExp(RegExpComposer.sanitizeGroups(EnglishDateTime.NumberCombinedWithDurationUnit));
   static final AnUnitRegex = RegExp(RegExpComposer.sanitizeGroups(EnglishDateTime.AnUnitRegex));
   static final DuringRegex = RegExp(EnglishDateTime.DuringRegex);
-  static final AllRegex = RegExp(EnglishDateTime.AllRegex);
+  static final AllRegex = RegExpComposer.sanitizeGroupsAndCompile(EnglishDateTime.AllRegex);
```

Comment:
Is there a reason that we would only want to sanitize a single regex? What about the others?

Reply:
@matthew-carroll Updated to sanitize all regexes.